src/consolebundle/CommandManager.py

```diff
@@ -1,17 +1,48 @@
-from typing import List
+from typing import List, Union, Tuple
 from consolebundle.ConsoleCommand import ConsoleCommand
 
 
 class CommandManager:
+
+    POSSIBLE_SEPARATORS = [':', ' ']
+
     def __init__(self, commands: List[ConsoleCommand]):
         self.__commands = commands
```

Comment:
Creating i map of (norrmalized) command names and commands here in __init__ would simplify the whole code a lot...

Reply:
Hmm. I cannot seem to see the simplification. Could you please elaborate on this?